src/generic/UnivPoly.jl

```diff
@@ -1170,6 +1170,7 @@ x*y - z
 ```
 """
 function universal_polynomial_ring(R::Ring; cached::Bool=true, internal_ordering::Symbol=:lex)
+   @req !is_trivial(R) "Zero rings are currently not supported as coefficient ring."
```

Comment:
Having a period at the end is somewhat rare in our error messages. A quick experiment in Oscar.jl:
```
$ git grep -E 'error\(.*\."' | wc -l
     101
$ git grep -E 'error\(' | wc -l
    1526
```
and
```
$ git grep -E '@req.*\."' | wc -l
     137
$ git grep -E '@req.*"' | wc -l
    1599
```
So less than 10% of our `@req calls use a trailing period. The numbers for `error` are a bit less reliably because they miss the case were the string spans over more than one line but I don't think there will be a fundamentally different outcome.

Anyway I am merging this and the other PRs now as this is a very minor point and also a matter of taste.